conf.d/pure.fish

```diff
@@ -97,5 +97,6 @@ _pure_set_default pure_symbol_ssh_prefix "" # suggestion: 'ssh:/' or '🔗🔐
 # Display Kubernetes/k8s context and namespace
 _pure_set_default pure_enable_k8s false
 _pure_set_default pure_symbol_k8s_prefix "☸" # ☸️
+_pure_set_default pure_color_k8s_symbol pure_color_dark
```

Comment:
Please rename the variable as `pure_color_k8s_prefix` (cf. [CONTRIBUTING.md](https://github.com/pure-fish/pure/blob/master/CONTRIBUTING.md#features-variables)) as the color setting target the _prefix_ of the k8s feature. Mixing color and symbols keyword would be confusing.
Note that, we already have similar cases for git features:

```
pure_symbol_git_unpushed_commits "⇡"
pure_color_git_unpushed_commits pure_color_info
```

```suggestion
_pure_set_default pure_color_k8s_prefix pure_color_dark
```
N.B.: Other files need updating

Reply:
Haha, yeah, that is indeed a bit confusing! Changed it